controllers/provider-alicloud/charts/provider-alicloud/templates/secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
```

Comment:
If .Values.config.machineImageOwnerSecret is not set, let's not create the secret.